### ISSUE TITLE
MSBUILD : error MSB1011: Specify which project or solution file to use because this folder contains more than one project or solution file

### ISSUE THREAD

**Issue by chenzhitong** [neo-project/neo-devpack-dotnet#547]
When the solution file and the project file are in the same folder and have different names, the following error will be reported
```
MSBUILD : error MSB1011: Specify which project or solution file to use because this folder contains more than one project or solution file
```

![image](https://user-images.githubusercontent.com/6295602/114332474-e24cd900-9b78-11eb-9fed-3476d95df84c.png)

Change their file names to the same name to compile successfully.

![image](https://user-images.githubusercontent.com/6295602/114332456-dd882500-9b78-11eb-854b-135d1d2a97c0.png)

**Comment by erikzhang** [comment]
It should be fixed by #546

**Comment by chenzhitong** [comment]
Yes, it's fixed
